File: app/src/main/java/com/example/purepawapp/ui/profile/ContactFragment.java
package com.example.purepawapp.ui.profile;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.navigation.fragment.NavHostFragment;

import com.example.purepawapp.databinding.FragmentContactBinding;
import com.example.purepawapp.ui.common.BaseFragment;

public class ContactFragment extends BaseFragment<FragmentContactBinding> {

    public ContactFragment() {
        super(FragmentContactBinding::inflate);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        getBinding().btnBack.setOnClickListener(v -> NavHostFragment.findNavController(this).popBackStack());

        getBinding().btnSend.setOnClickListener(v -> {
            getBinding().etName.getText().clear();
            getBinding().etEmail.getText().clear();
            getBinding().etMessage.getText().clear();
            Toast.makeText(requireContext(), "Đã gửi tin nhắn!", Toast.LENGTH_SHORT).show();
        });
    }
}

File: app/src/main/java/com/example/purepawapp/ui/profile/ContactFragment.kt
package com.example.purepawapp.ui.profile

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.navigation.fragment.findNavController
import com.example.purepawapp.databinding.FragmentContactBinding
import com.example.purepawapp.ui.common.BaseFragment

class ContactFragment : BaseFragment<FragmentContactBinding>(FragmentContactBinding::inflate) {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        binding.btnBack.setOnClickListener {
            findNavController().popBackStack()
        }

        binding.btnSend.setOnClickListener {
            binding.etName.text?.clear()
            binding.etEmail.text?.clear()
            binding.etMessage.text?.clear()
            Toast.makeText(requireContext(), "Đã gửi tin nhắn!", Toast.LENGTH_SHORT).show()
        }
    }
}
